### PR TITLE
#43 feat: add Launch dev script

### LINKED ISSUE #43: Set up launch scripts
Blocker: launch scripts urls to be provided by @tim-white-esri for each env

### PR DESCRIPTION
**Pull request by tim-white-esri**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #43 

Test URLs:
- Original: https://www.esri.com/en-us/about/about-esri/overview
- Before: https://main--esri--aemsites.hlx.live/en-us/about/about-esri/overview
- After: https://launch--esri--aemsites.hlx.live/en-us/about/about-esri/overview

### CODE REVIEW

**Comment by alexcarol** [comment]
Can you move this to delayed.js?

**Comment by tim-white-esri** [comment]
> Can you move this to delayed.js?

Done.

**Comment by tim-white-esri** [comment]
Now it's done

File: scripts/delayed.js [hunk 1]
```diff
@@ -5,3 +5,9 @@ import { sampleRUM } from './aem.js';
 sampleRUM('cwv');
 
 // add more delayed functionality here
+
+// Launch script
+const script = document.createElement('script');
+script.src = 'https://assets.adobedtm.com/launch-ENd3e1f5f2e5b44e4f8c9e7b0c4a2f8b3.min.js';
```
Comment (alexcarol) [inline scripts/delayed.js:11]: Can you use loadScript instead? The code will be much simpler for it.

Comment (tim-white-esri) [reply scripts/delayed.js]: Done. Appreciate the feedback.

File: scripts/delayed.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
-import { sampleRUM } from './aem.js';
+import { sampleRUM, loadScript } from './aem.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
 // add more delayed functionality here
+
+// Launch script
+loadScript('https://assets.adobedtm.com/2d251f50426c/e52f833be42a/launch-bdb68bbb4cf5-development.min.js', () => {
```
Comment (alexcarol) [inline scripts/delayed.js:10]: If you want to pass an empty second parameter (for attrs) you can just pass null or not pass anything at all so it defaults to undefined `loadScript('...');`

Comment (tim-white-esri) [reply scripts/delayed.js]: First attempt at using the loadScript. Updated.

**Review by alexcarol — APPROVED** [review]

**Pull request merged**